Marlin/src/HAL/NATIVE_SIM/u8g/u8g_com_st7920_sw_spi.cpp

```diff
@@ -71,13 +71,13 @@ static uint8_t SPI_speed = 0;
 
 static uint8_t swSpiTransfer(uint8_t b, const uint8_t spi_speed, const pin_t sck_pin, const pin_t miso_pin, const pin_t mosi_pin) {
```

Comment:
Perhaps this should be renamed to `swSpiTransfer_mode_3` to explicitly indicate that it is using Mode 3 and not Mode 0.